src/python/pants/backend/project_info/dependencies.py

```diff
@@ -75,48 +89,36 @@ class Dependencies(Goal):
 async def dependencies(
     console: Console, addresses: Addresses, dependencies_subsystem: DependenciesSubsystem
 ) -> Dependencies:
-    if dependencies_subsystem.transitive:
-        transitive_targets = await Get(
-            TransitiveTargets, TransitiveTargetsRequest(addresses, include_special_cased_deps=True)
-        )
-        targets = Targets(transitive_targets.dependencies)
-    else:
-        target_roots = await Get(UnexpandedTargets, Addresses, addresses)
-        dependencies_per_target_root = await MultiGet(
-            Get(
-                Targets,
-                DependenciesRequest(tgt.get(DependenciesField), include_special_cased_deps=True),
-            )
-            for tgt in target_roots
-        )
-        targets = Targets(itertools.chain.from_iterable(dependencies_per_target_root))
-
-    include_source = dependencies_subsystem.type in [
-        DependencyType.SOURCE,
-        DependencyType.SOURCE_AND_THIRD_PARTY,
-    ]
-    include_3rdparty = dependencies_subsystem.type in [
-        DependencyType.THIRD_PARTY,
-        DependencyType.SOURCE_AND_THIRD_PARTY,
-    ]
-
-    address_strings = set()
-    third_party_requirements: Set[str] = set()
-    for tgt in targets:
-        if include_source:
-            address_strings.add(tgt.address.spec)
-        if include_3rdparty:
-            if tgt.has_field(PythonRequirementsField):
-                third_party_requirements.update(
-                    str(python_req) for python_req in tgt[PythonRequirementsField].value
-                )
-
-    with dependencies_subsystem.line_oriented(console) as print_stdout:
-        for address in sorted(address_strings):
-            print_stdout(address)
-        for requirement_string in sorted(third_party_requirements):
-            print_stdout(requirement_string)
-
+    depgraph = await Get(
+        DependencyGraph, DependencyGraphRequest(addresses, dependencies_subsystem.transitive)
```

Comment:
Non-blocking: the `dependees` goal could likely use this same graph, and then reverse it.

Reply:
My plan is to retcon dependees to be entirely symmetric with dependencies.